pkgs/servers/web-apps/livebook/default.nix

```diff
@@ -0,0 +1,45 @@
+# Created with inspiration from https://github.com/hauleth/nix-elixir/blob/master/pkgs/livebook.nix
+
+{ lib, beamPackages, makeWrapper, rebar3, elixir, erlang, fetchFromGitHub }:
+beamPackages.mixRelease rec {
+  pname = "livebook";
+  version = "0.9.2";
+
+  inherit elixir;
+
+  buildInputs = [ erlang ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "livebook-dev";
+    repo = "livebook";
+    rev = "v${version}";
+    hash = "sha256-khC3gtRvywgAY6qHslZgAV3kmziJgKhdCB8CDg/HkIU=";
+  };
+
+  mixFodDeps = beamPackages.fetchMixDeps {
+    pname = "mix-deps-${pname}";
+    inherit src version;
+    sha256 = "rwWGs4fGeuyV6BBFgCyyDwKf/YLgs1wY0xnHYy8iioE=";
```

Comment:
```suggestion
    hash = "sha256-rwWGs4fGeuyV6BBFgCyyDwKf/YLgs1wY0xnHYy8iioE=";
```

Reply:
Unfortunately, I don't think `fetchMixDeps` supports using `hash` instead of `sha256` right now. At least, this is the error I get when I try:

```
$ nix-build -A livebook
error: anonymous function at /home/munksgaard/src/nixpkgs/pkgs/development/beam-modules/fetch-mix-deps.nix:3:1 called without required argument 'sha256'

       at /home/munksgaard/src/nixpkgs/pkgs/servers/web-apps/livebook/default.nix:21:16:

           20|
           21|   mixFodDeps = beamPackages.fetchMixDeps {
             |                ^
           22|     pname = "mix-deps-${pname}";
(use '--show-trace' to show detailed location information)
```

Reply:
I had a go at this https://github.com/NixOS/nixpkgs/pull/235733 , I'm not sure what is the backward compatibility policy toward the change.